src/communication.rs

```diff
@@ -69,6 +70,18 @@ impl PartialEq for Sender {
     }
 }
 
+// Fixes #255.
+impl Eq for Sender { }
```

Comment:
Couldn't this be derived ?

Reply:
Unfortunately not. Deriving `Eq` seems to currently require that all fields also implement `Eq`, which doesn't hold for `mio::channel::SyncSender<Command>`.

Reply:
Oh, I thought deriving Eq only needed the type to be `PartialEq`, guess not

Reply:
I was surprised by that as well and [played with it](https://play.integer32.com/?version=stable&mode=debug&edition=2018&gist=c0d63a2f235507d06bd396eacf1f9ec7). Manually implementing works as expected though.